### DISCUSSION TITLE
Scan results from ec2 instance to Stig Manager Collections

### DISCUSSION THREAD

**Discussion by hasib-haque** [NUWCDIVNPT/stig-manager#1305] [Show and tell]
I have xccdf scan results in a S3 bucket which I have mounted on the same ec2 instance where I am running the stigmanager containers.
I have made the mounted shares available as volumes to the api container, but the contents don't show up as collections in STIG manager GUI. What do I need to do to make this work?

**Comment by cd-rite** [comment]
Hi @hasib-haque STIG Manager does not store or look at result files locally.  You would need to import your scan files into a STIGMan Collection via a client (for example, using our GUI) which will parse and process them, and make proper requests to the API to create Assets and Reviews as needed, storing the data in the MySQL database.
https://stig-manager.readthedocs.io/en/latest/user-guide/user-guide.html#collection-builder
